File: 3-python/s5_funciones_01.py
#IMPORTAR FUNCION
import math

#DEFINIR FUNCION Y PROCESO Y RETORNO
def area_circulo(radio):
    area = math.pi * (radio ** 2)
    return area

#SOLICITO AL USUARIO VALOR
radio=float(input("Ingrese radio:"))

#LLAMO LA FUNCIO Y GENERO RESULTADO
resultado = area_circulo(radio)

#IMPRIMO EL RESULTADO
print(f"Area circulo : {resultado}")
print("Area circulo :",resultado)

File: 2-c#/s5_funciones_01.cs
//DECLARAR VARIABLE
double radio=0;
double resultado=0;

//OBTENER LOS VALORES DEL USUARIO
Console.WriteLine("Ingrese radio:");
radio = double.Parse(Console.ReadLine());

//LLAMAR FUNCION Y PASO VALOR
resultado = area_circulo(radio); 

//MOSTRAR RESULTADO DE LA FUNCION
Console.WriteLine("Radio es:" + resultado);

//CREAR FUNCION
//public double radio_circulo(double radio)
double area_circulo(double radio)
{
    //return area = 3.14159 * radio ^ 2;
    double resultado=0;
    resultado = Math.PI * Math.Pow(radio,2);
    return resultado;
}